docs/src/tutorials/custom_impurity_density_pn_junction_lit.jl

```diff
@@ -22,18 +22,19 @@ using SolidStateDetectors
 using Unitful
 
 sim = Simulation{Float32}(SSD_examples[:InfiniteParallelPlateCapacitor])
-plot(sim.detector)
+plot( sim.detector.semiconductor, fillalpha = 0.4)
+plot!(sim.detector, xlims = (-0.006, 0.006), aspect_ratio = :none)
 
 # ## Define the Impurity Density
 
-# Now, we define the model for impurity density of the p-n junction.
+# Now, we define the model for the impurity density of the p-n junction.
 # In order to do so, we need two define two things.
 
 # 1. We have to define a `struct` which has to be 
 # subtype of `SolidStateDetectors.AbstractImpurityDensity{T}`.
 # We define the model with two parameters in order to be able to vary the density level 
-# on the p-type and n-type regions independently. 
-# More parameters could be added. E.g., the position of the p-n junction.
+# in the p-type and n-type regions independently. 
+# More parameters could be added, e.g., the position of the p-n junction.
```

Comment:
I think we don't need the comma after e.g. but I might be wrong.

Reply:
I just looked it and it seems to be another British vs American English thing...